packages/openapi-v3/src/__tests__/unit/spec-contributor/fixtures/info.spec.extension.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {bind} from '@loopback/context';
+import {
+  asSpecEnhancer,
+  OAISpecEnhancer,
+} from '../../../../extension-point/types';
+import {OpenApiSpec} from '../../../../types';
+
+/**
+ * A spec contributor to add OpenAPI info spec
+ */
+@bind(asSpecEnhancer)
+export class InfoSpecEnhancer implements OAISpecEnhancer {
+  modifySpec(spec: OpenApiSpec) {
+    spec.info = {
+      title: 'LoopBack Test Application',
+      version: '1.0.1',
+    };
+    return spec;
+  }
```

Comment:
I would prefer the framework to provide a tool for merging OpenAPI spec fragments for us, so that enhancers don't have to (re)implement the logic.

```ts
@bind(asSpecEnhancer)
export class InfoSpecEnhancer implements OAISpecEnhancer {
  modifySpec(spec: OpenApiSpec) {
    return assignSpecFragment(spec, {
      info: {
        title: 'LoopBack Test Application',
        version: '1.0.1',
      },
    });
  }
}
```